src/app/components/ResultProducts/ResultProducts.stories.tsx

```diff
@@ -1,25 +1,26 @@
-import React from 'react';
+// import React from 'react';
 import ResultProducts from './ResultProducts';
 
 export default {
   title: 'Component/ResultProducts',
   component: ResultProducts,
 };
 
-export const ResultAirpods = (): JSX.Element => (
-  <ResultProducts
-    imgSrc="https://appletoolbox.com/wp-content/uploads/2019/10/Airpods-pro-3.jpg"
-    title="Apple Airpods 2. Generation"
-    vendor="ebay"
-    price="ab 148 €"
-  />
-);
+// export const ResultAirpods = (): JSX.Element => (
+//   <ResultProducts
+//     id="Hi"
+//     image="https://cdn.idealo.com/folder/Product/6837/6/68376…/s1_produktbild_mittelgross/apple-airpods-pro.jpg"
+//     title="Apple Airpods 2. Generation"
+//     offers="83 Angebote"
+//     price="ab 148 €"
+//   />
+// );
```

Comment:
Why did you comment this out?
If there is an issue, better fix it before merging this pull request.

Reply:
Storybook had been causing problems after I added another type in my data type. I am still trying to figure out what cause the problem

Reply:
Feel free to share the error message in Slack or https://github.com/neuefische/help/issues to get help about the problem.